src/react-apps/applications/ux-editor/src/components/config/EditModalContent.tsx

```diff
@@ -197,6 +212,14 @@ class EditModalContentComponent extends React.Component<IEditModalContentProps,
     );
   }
 
+  public renderPropertyLabel(textKey: string) {
+    return (
+      <Typography gutterBottom={false} className={this.props.classes.inputHelper}>
```

Comment:
gutterBottom is false by default.

Reply:
Yep, but we have tslint rules that say we should always explicitly set parameters in JSXElement expressions.